src/services/services.ts

```diff
@@ -3705,6 +3732,41 @@ namespace ts {
                 return entries;
             }
 
+            function getJsDocCompletionEntries(fileName: string, position: number): CompletionEntry[] {
+                let sourceFile = getValidSourceFile(fileName);
+                let hasJsDocComment = ts.hasDocComment(sourceFile, position);
+                if (!hasJsDocComment) {
+                    return undefined;
+                }
+
+                // The current position is right after an At sign
+                if (sourceFile.text.charCodeAt(position - 1) === CharacterCodes.at) {
+                    return getAllJsDocCompletionEntries();
+                }

```

Comment:
The reason is I didn't see anywhere else in the code compares chars directly, so I thought that was out of some sort of consideration.